build.assets/tooling/go.mod

```diff
@@ -49,4 +49,35 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
 
+require (
+	github.com/Masterminds/sprig/v3 v3.2.3
+	github.com/gogo/protobuf v1.3.2
+	k8s.io/apiextensions-apiserver v0.27.4
+)
+
+require (
```

Comment:
Please merge required into an already existing section instead of creating a new one.